frontend/src/app/login/login.drv.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Directive for login formular
+ */
+angular.module('flocs.user')
+.directive('loginForm', function(){
+	return{
+		restrict:'EA',
+		template:
```

Comment:
put the template in a file, most likely into login.tpl.html for this html and use reference to the file here